### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: TraceScreen.c [hunk 1]
```diff
@@ -93,8 +93,7 @@ bool TraceScreen_forkTracer(TraceScreen* this) {
 
       // Should never reach here, unless execlp fails ...
       const char* message = "Could not execute 'strace'. Please make sure it is available in your $PATH.";
-      ssize_t written = write(STDERR_FILENO, message, strlen(message));
-      (void) written;
+      (void)! write(STDERR_FILENO, message, strlen(message));
```
Comment (BenBE) [inline TraceScreen.c:96]: Why negate the result from `write`?

**Review by BenBE — CHANGES_REQUESTED** [review]

File: HostnameMeter.c [hunk 2]
```diff
@@ -19,8 +19,7 @@ static const int HostnameMeter_attributes[] = {
    HOSTNAME
 };
 
-static void HostnameMeter_updateValues(Meter* this, char* buffer, size_t size) {
-   (void) this;
+static void HostnameMeter_updateValues(ATTR_UNUSED Meter* this, char* buffer, size_t size) {
```
Comment (natoscott) [inline HostnameMeter.c:22]: @cgzones this is fixed in #553 FWIW.

File: TraceScreen.c (hunk 1 above)
Comment (cgzones) [reply TraceScreen.c:96]: `write(2)` is annotated with `__attribute__((warn_unused_result)` so compilers warn on ignored return value.
Casting the result to `(void)` only pleases clang but not gcc, see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66425.
See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66425#c34 for the negation hack.

Comment (BenBE) [reply TraceScreen.c:96]: Urgs, k.

**Review by BenBE — APPROVED** [review]

**Pull request merged**